packages/platform/examples/base/src/pages/event-types.tsx

```diff
@@ -95,7 +95,161 @@ export default function Bookings(props: { calUsername: string; calEmail: string
         {eventTypeId && (
           <div>
             <EventTypeSettings
-              customClassNames={{ atomsWrapper: "!w-[50vw] !m-auto" }}
+              customClassNames={{
+                atomsWrapper: "!w-[50vw] !m-auto",
+                eventAdvancedTab: {
+                  destinationCalendar: {
+                    container: "bg-purple-50 border border-purple-200 shadow-md",
+                    select: "border border-purple-300 shadow-sm",
+                    innerClassNames: {
+                      input: "",
+                      option: "",
+                      control: "",
+                      singleValue: "",
+                      valueContainer: "",
+                      multiValue: "",
+                      menu: "",
+                      menuList: "",
+                    },
+                    label: "",
+                  },
+                  eventName: {
+                    container: "bg-purple-50 border border-purple-200 shadow-sm",
+                    input: "border border-purple-300 shadow-inner",
+                    label: "",
+                    addOn: "",
+                  },
+                  addToCalendarEmailOrganizer: {
+                    switch: {
```

Comment:
Is there a reason `switch` is an object with container in it and other places we have `switchContainer` ?

Reply:
![Screenshot 2024-11-21 at 5 34 58 PM](https://github.com/user-attachments/assets/c7ded9b4-c7dc-4f67-b2c3-36cc4e8f990f)
fixed